Tests/OptimizelyTests-Common/LoggerTests.swift

```diff
@@ -18,36 +18,159 @@ import XCTest
 
 class LoggerTests: XCTestCase {
 
-    func testDebugLog() {
-        CustomLogger.logCount = 0
-        let logger = CustomLogger()
-        CustomLogger.logLevel = .debug
+    let logger = TestLogger()
+    
+    func testOPTLogger_DefaultMethods() {
+        // String messages
+        
+        logger.e("Error Message")
```

Comment:
since most of these tests are very similar, can we have them in a loop? there's alot of repetitive code here.

Reply:
this is an overall suggestion.